cypress/support/e2e.js

```diff
@@ -5,3 +5,28 @@
 
 import '../utils/commands';
 import '../utils/apps/commands';
+import '../utils/dashboards/workspace-plugin/commands';
```

Comment:
imo the commands should be exported in `dashboards/commands` and then we just import it instead of nesting a plugin